parser/ndt.go

```diff
@@ -15,10 +16,10 @@ type NDTParser struct {
 	tmpDir string
 }
 
-func (n *NDTParser) Parse(fn string, table string, rawSnapLog []byte) (interface{}, error) {
+func (n *NDTParser) Parse(meta map[string]bigquery.Value, testName string, table string, rawSnapLog []byte) (interface{}, error) {
```

Comment:
Add a TODO: make TableID as global const var, instead of passing it as input param everywhere.